src/script/components/app-sidebar.ts

```diff
@@ -2,10 +2,87 @@ import { LitElement, css, html, customElement } from 'lit-element';
 
 @customElement('app-sidebar')
 export class AppSidebar extends LitElement {
-
   static get styles() {
     return css`
-      
+      aside.sidebar {
+        color: #ffffff;
+        background: var(--primary-color);
+        height: 100vh;
+        width: 280px;
+        position: fixed;
+        display: flex;
+        flex-direction: column;
+        place-items: center;
+        overflow-x: auto;
+      }
+      aside.sidebar img {
+        width: calc(100% - 150px);
+        margin: 1rem auto;
+      }
+      aside.sidebar fast-menu {
+        width: 280px;
+        margin: 0;
+        padding: 0;
+        border: none;
+        border-radius: 0;
+        box-shadow: none;
+      }
+      aside.sidebar h1,
```

Comment:
These styles, along with the font styles currently in globals.css should probably be split out into a seperate CSS file (like with animations.css) and imported where needed.

Reply:
I'll add that